clusterctl/cmd/create_cluster.go

```diff
@@ -144,9 +144,27 @@ func getProvider(provider string) (clusterdeployer.ProviderDeployer, error) {
 	case "google":
 		return google.NewMachineActuator(google.MachineActuatorParams{})
 	case "terraform":
```

Comment:
Do you plan to have a separate option for the vsphere provider, or do you think this will eventually be replaced by that one? #233

Reply:
I swapped for vsphere one. We can add terraform back should we having a case for that.